config/v1/types_authentication.go

```diff
@@ -67,6 +67,14 @@ type AuthenticationSpec struct {
 	// duration.
 	// +optional
 	ServiceAccountIssuer string `json:"serviceAccountIssuer"`
+
+	// trustedServiceAccountIssuer is the identifier of the bound service account token issuer the server will trust.
```

Comment:
// trustedServiceAccountIssuer is an additional Service Account Issuer that the kube-apiserver will tolerate besides the issuer from the ServiceAccountIssuer.